traffic-participants.rst

```diff
@@ -0,0 +1,90 @@
+
+Use cases for traffic participants
+----------------------------------
+
+Different models may be involved in modeling a traffic participant.
+In all the use cases, a simulator loads and interprets a scenario and a map prior to execution.
+The scenario is, for example, provided by OpenSCENARIO.
+The map data is, for example, provided by OpenDRIVE.
+During runtime the simulator interacts with the traffic participants via OSI messages.
+There may be multiple instances of a traffic participant.
+The traffic participants are co-simulated.
+
+The following figure shows a very simple use case.
+
+.. image:: _static/images/osi-traffic-participant-use-case-1.png
+   :align: center
+   :alt: Simple traffic participant
+
+The traffic participant bases its behavior only on an idealized view of the area around it.
+If they exist, the traffic participant’s dynamics are included in the model.
+
+The following figure shows a traffic participant with separately modeled behavior and dynamics.
+
+.. image:: _static/images/osi-traffic-participant-use-case-2.png
+   :align: center
+   :alt: Traffic participants with separate dynamics
+
+OSI currently provides only limited support for data structures describing measured internal states of the traffic participant.
+Actuator intentions are currently not covered by OSI and must be handled with a different data description format.
+
+The following figure shows a more complex traffic participant.
+
+.. image:: _static/images/osi-traffic-participant-use-case-3.png
+   :align: center
+   :alt: Traffic participant with sensor models, AD function, and dynamics model
+
+This use case will probably be relevant for modeling the ego vehicle, which includes the system-under-test.
+The traffic participant includes an arbitrary number of sensor models
+The sensor models consume sensor view and produce sensor data.
+The AD function consumes sensor data and produces input for the dynamics model.
+OSI currently does not support data flow to dynamics models.
+The loop to the environment simulation is closed via traffic update.
+
+The following figure shows a cooperative use case with both an AD function and a human driver.
+
+.. image:: _static/images/osi-traffic-participant-use-case-4.png
+   :align: center
+   :alt: Traffic participant with an AD function and a human driver
+
+It is possible to model a traffic participant with an AD function in the loop, but a human driver can still override the actuation command.
+Such a cooperative use case is, for example, relevant to studies on human-machine interaction.
+In this example, a virtual on-screen representation of the scenario, or mock-up, is added after the AD function.
+The driver-in-the-loop interacts with the dynamics model via this mock-up.
+OSI's limitations regarding dynamics-model input apply in this example as well.
+
+The following figure shows a similar example with the difference that there is only a human driver in control.
```

Comment:
I would leave out this part as it is not enough supported by OSI and not really used (so far).
Focus for this can be in upcoming versions, but now it's not supported as well as needed.

Reply:
Does anyone object to deleting this part (for now)?
If not, I delete this part tomorrow (23.02.2021), including the image `osi-traffic-participant-use-case-5.png`.

Reply:
Removed section in commit  [43b37dd](https://github.com/OpenSimulationInterface/osi-documentation/pull/20/commits/43b37ddafc290438c3498a0036a6c0724ceb21d4).